src/pages/recipes/AddIngredient.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import * as yup from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { setLocale } from "yup";
+import { useNavigate } from "react-router-dom";
+import propTypes from "prop-types";
+import { Input } from "../../components/Input/Input";
+import { Container } from "../../components/Container/Container";
+import { Button } from "../../components/Button/Button";
+import { Text } from "../../components/Text/Text";
+import { ButtonGroup } from "../../components/ButtonGroup/ButtonGroup";
+
+function AddIngredient({ onSubmit }) {
+  setLocale({
```

Comment:
wouldn't it be better to set these globally? for example - create some file and import it in `main.tsx`, wdyt?